ufficio_soci/viste.py

```diff
@@ -1370,6 +1384,24 @@ def us_tesserini_emissione_processa(request, me):
 
             fine = True
 
+            # invio messaggio all'US richiedente se tesserino rifiutato
+            if(stato_richiesta == Tesserino.RIFIUTATO):
+                for i in tesserini:
+                    persona = Persona.objects.get(pk=i.persona.id)
+                    messaggio = Messaggio.costruisci(
```

Comment:
Ci sono un paio problemi con questo metodo per inviare un messaggio di posta. Per favore controlla la documentazione di Jorvik, alla pagina "[Posta: Inviare una e-mail](https://github.com/CroceRossaItaliana/jorvik/wiki/Posta:-Inviare-una-e-mail)".

- Il metodo `Messaggio.costruisci` non effettua l'invio del messaggio per email. Usa `Messaggio.costruisci_e_accoda`, come da documentazione.
- E' necessario creare un modello per il messaggio. Il testo del messaggio va nel modello, non qui. Questo ti permette di usare il sistema di templating di Django, piuttosto che fare concatenazione di stringhe (ugh!).
- `allegati=[]` non e' necessario.

Altri commenti non relativi strettamente al messaggio:
- Puoi sostituire:
  ```python
  destinatari=[Persona.objects.get(pk=i.richiesto_da_id)]
  ```
  Con, semplicemente:
  ```
  destinatari=[i.richiesto_da]
  ```
  Risparmiamo una round trip al database.
- In modo simile, 
  ```python
  persona = Persona.objects.get(pk=i.persona.id)
  ```
  Perche'? `i.persona` e' gia' un oggetto Persona. Non hai bisogno di ottenerne l'ID e cercarlo nuovamente in database.

- Cerca di rispettare PEP-8 quanto piu' possibile. Non usare le parentesi attorno alle condizioni, a meno che siano strettamente necessarie:
  ```python
  if(stato_richiesta == Tesserino.RIFIUTATO):
  ```
  Diventa:
  ```python
  if stato_richiesta == Tesserino.RIFIUTATO:
  ```
  PEP-8 e' molto importante! Puoi configurare il tuo IDE preferito (Sublime Text, Pycharm) per farlo **automaticamente** per te! :robot: 
- Preferisci nomi significativi per le variabili, anche se questo vuol dire usare nomi piu' lunghi. Ad esempio,
  ```python
  for tesserino in tesserini:
  ```
  e' piu' significativo di:
  ```python
  for i in tesserini:
  ```
  specialmente perche' `i` e' un nome che indica un contenuto numerico, e Python -non essendo un linguaggo tipizzato forte- potrebbe permetterti di utilizzarlo come tale, fino a che non esplode tutto in produzione. I nomi di variabile significativi ci aiutano a tenere Gaia inesplosa! :boom: